bazel-out

```diff
@@ -0,0 +1,3 @@
+this placeholder prevents bazel from writing a symlink here which points to its output dir.
+That symlink often causes tools that traverse the repo to get confused when they find lots of extra go files.
+The _bazel symlink points to the same thing, but go doesn't traverse into paths that start with _.
```

Comment:
Nit: `_bazel` is not a symlink, but `_bazel/out` is.

Reply:
Fixed.